src/app/features/test-record-summary/test-record-summary.component.spec.ts

```diff
@@ -2,16 +2,35 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { TestRecordSummaryComponent } from './test-record-summary.component';
 import { TestResultModel } from '../../models/test-result.model';
+import { TestType } from '@models/test-type.model';
 
 const fakeRecord: TestResultModel = {
   testResultId: 'test',
   systemNumber: 'test',
   testStartTimestamp: new Date(),
```

Comment:
If this is set to Date type, do 21 and 28 need to be the same? Currently string

Reply:
Model accepts string or date type :D

Reply:
I think this import might need to be:
`import { TestType } from '../../models/test-type.model';`

Reply:
Updated

Reply:
`@models/` is an alias for `models` folder so both are valid. you can see path aliases defined in `tsconfig.json` e.g. `"@models/*": ["src/app/models/*"],`

Reply:
Thanks for the clarification @mcadenas-bjss - if we can keep the 2 model imports the same, the either path is fine  :)